### DISCUSSION TITLE
<2024+Best Code for datasets>

### DISCUSSION THREAD

**Discussion by Fishyoo1** [community/community#131178] [Programming Help]
### Select Topic Area

Product Feedback

### Body

import requests
from bs4 import BeautifulSoup
import logging

def scrape_website(url):
    try:
        response = requests.get(url)
        response.raise_for_status()
        soup = BeautifulSoup(response.content, 'html.parser')
        return soup.prettify()
    except requests.RequestException as e:
        logging.error(f"Error scraping website {url}: {e}")
        return None

**Comment by ebndev** [comment]
Thanks for posting in the GitHub Community, @Fishyoo1! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion. If you provide more details about the question you're trying to solve- that might help someone give you a nudge in the right direction!
